prov/verbs/src/ep_rdm/verbs_tagged_ep_rdm_states.c

```diff
@@ -452,10 +460,10 @@ fi_ibv_rdm_copy_unexp_request(struct fi_ibv_rdm_request *request,
 	assert((request->state.eager == FI_IBV_STATE_EAGER_RECV_WAIT4RECV) || 
 	       (request->state.eager == FI_IBV_STATE_EAGER_RECV_CLAIMED));
 
-	VERBS_DBG(FI_LOG_EP_DATA, "found req: len = %d, eager_state = %s, rndv_state = %s \n",
-		unexp->len,
-		fi_ibv_rdm_req_eager_state_to_str(unexp->state.eager),
-		fi_ibv_rdm_req_rndv_state_to_str(unexp->state.rndv));
+	VERBS_DBG(FI_LOG_EP_DATA, "found req: len = %llu, eager_state = %s, rndv_state = %s \n",
```

Comment:
What about `PRIu64` instead of `llu`? Seems, it can help us to avoid casting to the `unsigned long long` type
Please, update the same comment for all places (if `uint64_t` then use `PRIu64` instead of `llu`).

Reply:
I've never liked these. I think they break the flow. If you really want them, I'll fix the patch.

Reply:
Agree with you, I don't like with those macros too, but I just want to avoid casting.
But seems casting is excessive there, because the macro `PRIu64` is just `llu` on my system. We can just use `%llu` instead of `%PRIu64`.

Reply:
uint64_t is (usually) unsigned long on 64 bits systems, but unsigned long long on 32 bits systems. Hence the need for the cast.

Reply:
Seems, only `PRIu64` can fix this problem :)

Reply:
It's a trade-off. PR updated.